Refit-Tests/InterfaceStubGenerator.cs

```diff
@@ -57,9 +57,9 @@ public void HasRefitHttpMethodAttributeSmokeTest()
             var fixture = new InterfaceStubGenerator();
 
             var input = file.GetRoot().DescendantNodes()
-                .OfType<InterfaceDeclarationSyntax>()
-                .SelectMany(i => i.Members.OfType<MethodDeclarationSyntax>())
-                .ToList();
+                            .OfType<InterfaceDeclarationSyntax>()
```

Comment:
We're clearly still missing the R# setting that stops it from indenting chained method calls like this. Would be good to clean that up so we don't get unnecessary changes like this.